students/ThanhNhan/lesson03/list_lab.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+#Series 1
+fruit =['Apples', 'Pears', 'Oranges', 'Peaches']
+print (fruit) #display a list of fruit
```

Comment:
You'll want to remove the space between print and the parentheses.  print(fruit)